tests/tests_certificate_runafter.yml

```diff
@@ -84,6 +85,9 @@
           assert:
             that: "'status: MONITORING' in result.stdout"
 
+        - name: test - ensure cockpit_port is configured for firewall and selinux
+          include_tasks: tasks/check_firewall_selinux.yml
```

Comment:
Please don't add this to *all* tests -- separation of concerns, this is unnecessary redundancy. The one place where it really matters is `tests_port.yml`, and perhaps `tests_default.yml` in addition, but none of the others.